style/style.css

```diff
@@ -0,0 +1,103 @@
+html,
+body,
+a {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  text-decoration: none;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,400;1,500;1,700&display=swap');
+
+h1,
+h2,
+h3,
+h4,
+p,
+a {
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #f7f7f9;
+  width: 100%;
+}
+
+.top-mob {
+  height: 718px;
+  background-image: url("../img/header-shapes-mobile.png");
+  background-color: white;
+  border-bottom-left-radius: 5rem;
+  border-top-left-radius: 2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
```

Comment:
- [ ] **[OPTIONAL]** Please use the `background-size: cover` and `background-repeat: no-repeat` if you would like to fix the background for the upcoming milestones of this project.

Reply:
Thanks for the tips.
added the CSS properties ` background-size: cover and background-repeat: no-repeat